test/performance/odp_sched_perf.c

```diff
@@ -821,6 +865,34 @@ static int create_queues(test_global_t *global)
 
 				data  = odp_packet_data(pkt);
 				words = odp_packet_seg_len(pkt) / 8;
+			} else {
+				odp_event_vector_t evv = odp_event_vector_alloc(pool);
+				odp_event_t *tbl;
+				odp_buffer_t buf;
+
+				if (evv == ODP_EVENT_VECTOR_INVALID) {
+					ODPH_ERR("Error: alloc failed %u/%u\n", i, j);
+					return -1;
+				}
+				odp_event_vector_tbl(evv, &tbl);
+
+				if (tbl == NULL) {
+					ODPH_ERR("Error: retrieval of event vector table failed\n");
+					return -1;
+				}
```

Comment:
This is a fast path API which is assumed to never fail, so no need to check `tbl` value.